reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -107,10 +107,10 @@ def __init__(self,
         # Instantiate clients for various Kubernetes REST APIs
         self._corev1api = k8s_client.CoreV1Api()
         self._versionapi = k8s_client.VersionApi()
-        self._extbetav1api = k8s_client.ExtensionsV1beta1Api()
         self._appsv1api = k8s_client.AppsV1Api()
         self._rbacauthorizationv1api = k8s_client.RbacAuthorizationV1Api()
         self._storagev1api = k8s_client.StorageV1Api()
+        self._networkingv1api = k8s_client.NetworkingV1beta1Api()
```

Comment:
I'm not sure if this is the right moment, but is it possible to re-use k8s API client from `reana-commons`?

Reply:
Once we move to a more streamlined way of deploying (such as Helm) this is going to go away, so if we wait for it to happen this will just disappear, which is great :)